app/helpers/application_helper.rb

```diff
@@ -283,14 +283,14 @@ def flot_bar_chart(name, xaxis_label, yaxis_label, data, options = {})
                 }.merge(options))
   end
 
-  def select_action_button(title, options = {}, *args)
+  def select_action_button(title, options = {}, always_multiple = nil, *args)
```

Comment:
maybe it could be one of `options`? you'd have to delete it before passing to `content_tag`, something like

```ruby
always_multiple = options.delete(:always_multiple)
```

or a safer variant with deep_cloning options first